tests/e2e/medic-conf-tests.js

```diff
@@ -0,0 +1,54 @@
+const _ = require('lodash');
+const {expect} = require('chai');
+const util = require('util');
+const { API_PORT }=require('../constants');
+const exec = util.promisify(require('child_process').exec);
+
+const actions = ['compile-app-settings','backup-app-settings','convert-app-forms',
+  'convert-collect-forms','convert-contact-forms', 'backup-all-forms','upload-app-forms','upload-collect-forms',
+  'upload-contact-forms', 'upload-resources','upload-branding','upload-partners','upload-custom-translations',
+  'upload-privacy-policies'];
+
+describe('medic-conf supported actions', () => {
+  beforeAll( () =>{
+    // Change the directory
+    try {
+      process.chdir('config/default');
+    } catch (err) {
+      console.error('no need to change directory');
+    }
+  });
+
+  afterAll(() =>{
+    // Change the directory back to cht-core
+    process.chdir('../..');
+  });
+
+  const runCommand = async (action) => {
+    const { stdout, stderr } = await exec(`medic-conf --url=http://admin:pass@localhost:${API_PORT} ${action} --force`);
+    if (stderr) {
+      console.error(`error: ${stderr}`);
+    }
+    return stdout;
+  };
+
+  it('should execute  upload-app-settings', async () => {
+    try {
+      const result = await runCommand('upload-app-settings');
+      expect(result).to.contain(`INFO Settings updated successfully`);
+    } catch (error) {
+      console.log(error.stdout);
```

Comment:
Catching the error here will also catch the error that expect is throwing, and not fail the test.
This is only happening because you're using `chai.expect` instead of jasmine. A caught failed assertion still fails the test if using jasmine's expect. 

Same goes for all the other tests.